src/Reflection/ReflectionType.php

```diff
@@ -90,7 +90,7 @@ public function __toString() : string
             case $this->type instanceof Types\Void_:
                 return 'void';
             default:
-                return \ltrim((string) $this->type, '\\');
+                return \ltrim((string)$this->type, '\\');
```

Comment:
This should be reverted: spacing after casts is welcome and required.

Reply:
Perfect! Fixed.